cockroachcloud/connect-to-your-cluster.md

```diff
@@ -78,19 +92,20 @@ Use the Console to authorize networks:
 
     <img src="{{ 'images/v19.2/cockroachcloud/connect-modal.png' | relative_url }}" alt="Connect to cluster" style="border:1px solid #eee;max-width:50%" />    
 
-2. From the **User** dropdown, select the SQL user you created in [Step 2. Create a SQL user](#step-2-create-a-sql-user).
-3. From the **Region** dropdown, select the region closest to where your client or application is running.
-4. From the **Database** dropdown, select the database you want to connect to.
+2. **IP Allowlist** is selected by default as the **Network Security** option. If you had [enabled VPC peering while creating your cluster](create-your-cluster.html#step-6-enable-vpc-peering-optional) for your GCP cluster and [establish a VPC Peering connection](#establish-a-vpc-peering-connection), select **VPC Peering** instead.
```

Comment:
I think this wants to be "established a VPC ..." since the other action is also past tense?

I think it might be clearer to revise this as something like

> Select *VPC Peering*  if you have already:
> 1. Enabled VPC peering ...
> 2. Established a VPC peering connection ...